### ISSUE TITLE
Website: rewrite "help" section

### ISSUE THREAD

**Issue by thaJeztah** [docker/docker#9300]
**TL;DR**
Clarify that GitHub is primarily meant for *issues* / *bugs*, not for general support questions.

I just wanted to point someone to IRC and the Google Groups, because those channels are more appropriate for support questions, but doing so noticed that the help section on the website is really not very clear on this.

From that page (https://docker.com/resources/help/):

> The Docker community is knowledgeable, large, and friendly. Please jump in, ask questions, and join the conversation.

(Followed by links to IRC, Google Groups, GitHub, StackOverflow and Twitter)

**Proposal**
To serve people better, and reduce the number of support questions on GitHub, I think it would be better to clarify which channels are meant for "what".

**note**
Ironically, I'm not sure that this is the right location for this issue, however, the https://github.com/docker/www.docker.io repository looks to be outdated and not actively maintained. *Is that repository still in use?* If not, maybe the issue tracker for that repository should be closed.

**Comment by thaJeztah** [comment]
@jfrazelle another issue that probably doesn't belong here; I think it's best to close it, but two things might be needed;

 - update the status and/or close down the https://github.com/docker/www.docker.io repository (is it still used or not?)
 - clarify where issues wrt the website should be posted

**Comment by jfrazelle** [comment]
ping @kencochrane ^^ :) thanks @thaJeztah

**Comment by thaJeztah** [comment]
For reference, I created an issue at support.docker.com to update the status of the mentioned repository;

https://support.docker.com/hc/en-us/requests/3115;

```
Close down/deprecate GitHub repository for docker.io

This issue came up a number of times on the Docker issue tracker;

The repository for the docker.io website (https://github.com/docker/www.docker.io) 
looks to be abandoned, but the issue tracker is still open and no "deprecation" 
message is present to clarify the state of the repository.

I discussed this with Docker maintainer Jessie Frazelle, and she thought the best 
way to deal with this is to open an issue at Docker support.

Would it be possible to;

- Add a "deprecation warning" (or some message explaining the state of things) to 
  the repositories README.md 
- Add a message to point people to the right channel for reporting issues w.r.t. the 
  docker.com/docker.io and registry websites 
- Close the issue tracker (people are currently creating issues without getting a 
  response, which looks really bad)

Please let me know if you need additional input from my side. It would be 
appreciated to "keep me in the loop", so that I can help directing people to the
right channels for support/issues.

regards,

Sebastiaan van Stijn (@thaJeztah)
```

**Comment by jfrazelle** [comment]
so do we close here ?

**Comment by thaJeztah** [comment]
Yes, I waited for a response before closing, and got this;

>Hello,

>This is certainly something that has been discussed internally from time to time. I'll see if I can get the right folks to start closing it down more explicitly.

>Thanks!

>Jeff


**closing**, because this part of the website is not under control of the Docker "project", so unfortunate, there's nothing that can be done here.

(If someone *does* have ways to get this updated, feel free to comment)
